### PR TITLE
Add List function

### PR DESCRIPTION
**Pull request by aqblanco**
Added a list function that parses the `7za l` command output, and returns an array with the command lines as objects.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -35,11 +44,66 @@ function cmd(paramsArr, cb) {
...
     cb = onceify(cb);
     const proc = spawn(bin, args);
+    let output = '';
     proc.on('error', function (err) {
         cb(err);
     });
     proc.on('exit', function (code) {
-        cb(code ? new Error('Exited with code ' + code) : null);
+        let result = null;
+        if(args[0] = 'l') {
```
Comment (onikienko) [inline index.js]: @aqblanco 
Maybe `(args[0] === 'l')`. May you check?

**Comment by onikienko** [comment]
Hi @aqblanco  
Thank you for PR    
May you check my comment? https://github.com/onikienko/7zip-min/pull/1/commits/4619027eeaf5453409a2439ca5fe8826d7aec12b#r317590668

**Comment by onikienko** [comment]
May you also write a test for `list`? 
It is `test/test.js` file.  
After `test.serial('pack', async t => {......})` function you may add `test.serial('list', async t => {.....})` to test your `list` function.  

Thank you!

**Comment by onikienko** [comment]
@aqblanco  Thanks for fixing!  
Do you plan to add test for list command?  Then I will be able to merge

**Comment by aqblanco** [comment]
Yes, but first I need to figure out how to do the whole create a zipped file with contents to be able to list them. Haven't been able to look at it yet

**Comment by onikienko** [comment]
@aqblanco 
Cool. In the test.js there is `test.serial('pack', async t => {......})` function we create a .zip with one file.    
You may test after this function. Use `distPath` - it will contain .zip

**Comment by aqblanco** [comment]
Cool, will do it later today/this week, thanks for the tip!

**Comment by aqblanco** [comment]
Could not find a reliable way of testing the date and time fields :(

**Comment by onikienko** [comment]
I feel we may live without date and time in test :) Will merge and maybe update tests a little. It fails for me on size.

Thank you for your contribution!

**Pull request merged**